src/sections/community/Community.Suborgs.tsx

```diff
@@ -13,43 +14,188 @@ import AILogo from "public/assets/logos/acm-ai.svg";
+import SPACE2 from "public/assets/communities-images/SPACE/SPACE2.jpg";
+import SPACE3 from "public/assets/communities-images/SPACE/SPACE3.jpg";
+import SPACE4 from "public/assets/communities-images/SPACE/SPACE4.jpg";
+
+import Image from "next/image";
+
+
+const CommunityImages: React.FC<{
+  srcs: string[];
+  alts: string[];
+}> = ({ srcs, alts }) => {
+  return(
+    <div className="community__image-grid">
+      <div className="nextImageHolder pic1">
+        <Image
+          src={srcs[0] ?? CatLong.src}
+          alt={alts[0] ?? `Half Cat 1`}
+          layout="fill"
+          />
+      </div>
+      
+      <div className="nextImageHolder pic2"
+      >
+        <Image
+        src={srcs[1] ?? CatLong.src}
+        alt={alts[1] ?? `Half Cat 2`}
+        layout="fill"
+          />
+      </div>
+      
+
+      <div className="nextImageHolder pic3">
+        <Image
+          src={srcs[2] ?? CatSquare.src}
+          alt={alts[2] ?? `Square Cat`}
+          layout="fill"
+          />
+      </div>
+      
+
+      <div className="nextImageHolder pic4">
+        <Image
+          src={srcs[3] ?? CatLong.src}
+          alt={alts[3] ?? `Super Long Cat`}
+          layout="fill"
+          />
+      </div>
+      
+    </div>
+  );
+};
+
+const CommunitiesTitle: React.FC<{
+  open?: boolean;
+  org: string;
+  color: string;
+  logo: string;
+  website: string;
+  glowClassName: string;
+  isMobile: boolean;
+  setSelected: (org: string) => void;
+}> = ({
+  open,
+  logo,
+  org,
+  color,
+  setSelected,
+}) => {
+
+  const getFullName = (org: string) => {
+    switch (org) {
+      case "AI":
+        return "Artificial Intelligence";
+      case "Cyber":
+        return "Cyber Security";
+      case "Hack":
+        return "Software Engineering";
+      case "Design":
+        return "Design";
+      case "Reach":
+        return "Outreach";
+      case "Space":
+        return "Space Career Fair";
+      default:
+        return org;
+    }
+  }
+
+  const [hover, setHover] = useState(false);
+
+  const filterDropShadow = `
+        drop-shadow(0px 0px 1px #FF0000) 
+        drop-shadow(0px 0px 1px #FF7F00)  
+        drop-shadow(0px 0px 1px #FFFF00)  
+        drop-shadow(0px 0px 1px #00FF00)  
+        drop-shadow(0px 0px 1px #0000FF)  
+        drop-shadow(0px 0px 1px #2E2B5F)  
+        drop-shadow(0px 0px 1px #8B00FF) 
+        `; 
+
+  return (
+    <>
+      <div className="community__sub-orgs__logo-card" data-open={open} >
+          <img
+            src={logo}
+            alt={`ACM ${org}`}
+            className={`community__sub-orgs__logo-card__logo ${((open || hover)) ? 'glow-on-hover' : ''}`}
+            onClick={() => setSelected(org)}
+            style={{'--shadow-color': color, filter: (open || hover) ? ` ${color=='' ? `` : `drop-shadow(0 0 1rem ${color})`} ` : 'none'} as React.CSSProperties}
+            onMouseEnter={() => setHover(true)}
+            onMouseLeave={() => setHover(false)}
```

Comment:
i feel like this could be done with CSS

Reply:
Can we make this a new ticket? This and [this commnet](https://github.com/acmucsd/main-website/pull/306#discussion_r1841497115)